lib/client-api/src/client_api.ts

```diff
@@ -129,6 +129,7 @@ export default class ClientApi {
     };
   };
 
+  // FIXME: not sure what to do about this API
```

Comment:
I guess this is a public API. Should I do something like:

```js
if (/* separators are set */) {
  return separators;
} else if (/* any kind has a - or a | and showRoots is undefined */) {
  return oldDefaults;
} else {
  return { hierarchySeparator: '/' };
}
```

Reply:
That makes sense to me